extensions/kubeapi/rbac/rbac.go

```diff
@@ -66,3 +72,32 @@ var RoleTemplateGroupVersionResource = schema.GroupVersionResource{
 	Version:  Version,
 	Resource: "roletemplates",
```

Comment:
Can we define this as a constant in rbac.go?

Reply:
this is only used once in the entire package so we can leave it as is. Are you in any of your tests going to use this in the rbac suite?

Reply:
I don't think I do. We can leave it as is